docs/user_guide/models/scvi.rst

```diff
@@ -0,0 +1,153 @@
+===========
+scVI
+===========
+
+**scVI** [#ref1]_ posits a flexible generative model of scRNA-seq count data that can subsequently
+be used for many common downstream tasks.
+
+The advantages of scVI are:
+
+    + Comprehensive in capabilities.
+
+    + Scalable to very large datasets (>1 million cells).
+
+The disadvantages of scVI include:
+
+    + Effectively requires a GPU for fast inference.
+
+    + Latent space is not interpretable, unlike that of a linear method.
+
+
+.. topic:: Tutorials:
+
+ - :doc:`/tutorials/notebooks/api_overview`
+ - :doc:`/tutorials/notebooks/harmonization`
+ - :doc:`/tutorials/notebooks/scarches_scvi_tools`
+ - :doc:`/tutorials/notebooks/scvi_in_R`
+
+
+Preliminaries
+==============
+scVI takes as input a scRNA-seq gene expression matrix :math:`X` with :math:`N` cells and :math:`G` genes.
+Additionally, a design matrix :math:`S` containing :math:`p` observed covariates, such as day, donor, etc, is an optional input.
+While :math:`S` can include both categorical covariates and continuous covariates, in the following, we assume it contains only one
+categorical covariate with :math:`K` categories, which represents the common case of having multiple batches of data.
+
+
+
+Generative process
+========================
+
+scVI posits that the observed UMI counts for cell :math:`n` and gene :math:`g`, :math:`x_{ng}`, are generated
+by the following process:
+
+.. math::
+   :nowrap:
+
+   \begin{align}
+    z_n &\sim {\mathrm{Normal}}\left( {0,I} \right) \\
+    \ell_n &\sim \mathrm{LogNormal}\left( \ell _\mu ,\ell _\sigma ^2 \right) \\
+    \rho _n &= f_w\left( z_n, s_n \right) \\
+    \pi_{ng} &= f_h^g(z_n, s_n) \\
+    x_{ng} &\sim \mathrm{ZeroInflatedNegativeBinomial}(\ell_n \rho_n, \theta_g, \pi_{ng})
```

Comment:
Here you might even write "ObservationModel" and say that it can be ZINB, NB or Poisson. Some people see ZINB and start criticizing before realizing things are extremely modular here